CHANGELOG.asciidoc

```diff
@@ -40,6 +40,7 @@ the agent will now include the database name in the dependency, thus `mysql/my-d
 addition, when using the `apm-agent-attach-cli-slim.jar`, which does not contain a bundled agent, the latest version will be downloaded
 from maven at runtime unless configured otherwise through  `--download-agent-version` - {pull}2659[#2659]
 * Added span-links to messaging systems instrumentation (supported by APM Server 8.3+ only) - {pull}2610[#2610]
+* Dubbo and OpenTelemetry bridge instrumentation are not experimental anymore and thus enabled by default - {pull}2632[#2632]
```

Comment:
made a lot more than these GA, we should put the full list here

Reply:
Good point, I'll clarify this with a bit more details here (not only what is now enabled by default).